src/library/librarycontrol.cpp

```diff
@@ -643,6 +643,15 @@ void LibraryControl::slotGoToItem(double v) {
     // Clear the search if the searchbox has focus
     emit clearSearchIfClearButtonHasFocus();
 
+    // If the focused window is a dialog, press Enter
+    auto focusWindow = QApplication::focusWindow();
+    if (focusWindow &&
+            focusWindow->type() == static_cast<Qt::WindowType>(Qt::Dialog)) {
```

Comment:
Don't use static_cast, test for the flags in the value returned by type(). I suppose we need to check for both Qt::Dialog and Qt::Popup.

Reply:
```
if (focusWindow && (focusWindow->type() & (Qt::Dialog | Qt::Popup))) {
```